safekeeper/src/lib.rs

```diff
@@ -69,7 +72,7 @@ pub struct SafeKeeperConf {
     pub wal_backup_enabled: bool,
     pub pg_auth: Option<Arc<JwtAuth>>,
     pub pg_tenant_only_auth: Option<Arc<JwtAuth>>,
```

Comment:
Hmm, why not put SwappableJwtAuth for the other params too? (pg_tenant_only_auth, pg_auth)

Reply:
I made this PR only for the pageserver. the safekeepers need more work than just putting `SwappableJwtAuth` to those items, they also need that HTTP endpoint for reloading for example. I did change the safekeepers because the safekeepers use some code that is shared with the pageserver.